### ISSUE TITLE
Activate RemoteWebDriver without "app" Desired Capability

### ISSUE THREAD

**Issue by koshdim** [2gis/Winium#22]
If I don't set "app" capability, this code
Activator.CreateInstance(typeof(RemoteWebDriver), new Uri("http://localhost:9999"), dc) as RemoteWebDriver;
throws an exception. is it possible to run without "app" being set?

**Comment by NickAb** [comment]
Are you running Winium.Desktop or Winium.StoreApps?

**Comment by alwayswanna** [comment]
Yeah, too have this problem. If i didnt start app in DesiredCapabilities, application is crash.
![image](https://user-images.githubusercontent.com/62200674/88203717-af1d8f00-cc63-11ea-9a35-154e4d5312b9.png)
